core/src/main/java/org/neo4j/ogm/session/delegates/DeleteDelegate.java

```diff
@@ -59,7 +60,8 @@ private DeleteStatements getDeleteStatementsBasedOnType(Class type) {
     private <T> void deleteAll(T object) {
         List<T> list;
```

Comment:
Can we rename this? `objects` like in `#deleteOneOrMoreObjects` or maybe `objectsForDeletion`?